spec/controllers/person/remarks_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+require 'spec_helper'
+
```

Comment:
die specs müssen unbedingt besser strukturiert werden. 
ich hatte dies bei deinem letzten PR noch aufgeräumt: https://github.com/hitobito/hitobito_sac_cas/blob/9eac6c0727b941bf83f1936ce69264a06aae3066/spec/jobs/qualifications/expiration_mailer_job_spec.rb
versuche losgelöst von der Implementierung zu denken, in Use Cases.

z.B. als Mitarbeiter kann ich alle Remarks lesen/bearbeiten
mach dir Gedanken welche Use Cases es alles gibt:
- Wenn ich Funktionär in einer andere Sektion bin habe ich beispielsweise auch keinen Zugriff auf die Person
- usw.